package.json

```diff
@@ -26,7 +26,6 @@
     "@atom/mocha-test-runner": "^1.6.0",
     "@types/atom": "^1.40.5",
     "@types/chai": "^4.2.14",
-    "@types/fuzzaldrin-plus": "0.6.1",
```

Comment:
Why remove the types? fuzzaldrin-plus-fast should support the exact same types.

Reply:
Because the native package has its own types. We don't need a separate installation anymore.